taxi/views.py

```diff
@@ -44,6 +65,26 @@
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_form.html"
+
+
+class CarUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_form.html"
+
+
+class CarDeleteView(LoginRequiredMixin, generic.edit.DeleteView):
+    model = Car
+    template_name = "taxi/car_confirm_delete.html"
```

Comment:
The same comment as above applies here. You don't need to override `template_name` if it's the same as the default one. The default `template_name` for a DeleteView is `<app name>/<model name>_confirm_delete.html`. So in this case, you don't need to specify `template_name` as `taxi/car_confirm_delete.html` because it's the default one.